src/Services/ServicesFactory.php

```diff
@@ -156,55 +175,43 @@ public function create( ...$service ) {
 
 	/**
 	 * @since 2.0
-	 *
-	 * @return SerializerFactory
```

Comment:
Why do we remove the comments? Just because typing is added doesn't equally mean that comments can be removed.

Reply:
Why duplicate what is already in the code in a comment? It makes the code more verbose / less information dense, makes it more costly to modify and adds room for inconsistencies.

Reply:
That is what the comment block is for to inform the user and document generator about what is expected here. Typing is added so the compiler and the PHP engine can verify against a codified expectation but that doesn't nullify the documentation task.

Reply:
Are we using a documentation generator that does not use key information added in PHP 7.0? I certainly hope not.

Reply:
> Are we using a documentation generator that does not use key information added in PHP 7.0? I certainly hope not.

I don't know but https://doc.semantic-mediawiki.org/ runs doxygen 1.8.13 that aside, when you are removing comments like above yet I'm adding comments (independent of whether new methods are typed or not) introduces inconsistent code styling.